.github/workflows/release-please.yaml

```diff
@@ -0,0 +1,82 @@
+name: Release Please
+
+on:
+  push:
+    branches:
+      - main
+      - 'release/v*'
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        description: 'Release version (e.g., 0.51.0)'
```

Comment:
IIUC we don't need to use `v` prefix for version, because PR titile format contains it:
https://github.com/aquasecurity/trivy/blob/efc7946b2e03619c6464e65d63c204960553c13f/release-please-config.json#L7
Perhaps we need to write about this to avoid human error.

Reply:
It's written in the doc, but it's a good idea also to add it here. Done. https://github.com/aquasecurity/trivy/pull/6795/commits/22abe912b02fd2376e114dd9313b8f18ed84efb6